scripts/lib/google-sheets/incentives-sheet-exporter.ts

```diff
@@ -95,22 +96,42 @@ export function collectedIncentiveToGoogleSheet(
   incentives: CollectedIncentive[],
   fieldMappings: AliasMap,
   useStandardHeader: boolean,
+  descriptionColumnName: string | undefined = undefined,
```

Comment:
The TS idiom for this is `descriptionColumnName?: string` -- it adds the `undefined` part implicitly

Reply:
Oops, not sure how I forgot about that.